src/App.jsx

```diff
@@ -14,70 +16,52 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const isSelectedGood = selectedGood ? `${selectedGood} is selected` : 'No goods selected';
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <h1 className="title is-flex is-align-items-center">
+        {isSelectedGood}
+        {selectedGood && (<button onClick={() => setSelectedGood('')} data-cy="ClearButton" type="button" className="delete ml-3" />)}
```

Comment:
I suggest moving all the attributes to a new line for better readability. It will make the code easier to understand)